src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -401,7 +401,7 @@
       text-decoration: underline;
 
       &:after {
-        font: normal normal normal 1em FontAwesome;
+        font: normal normal normal 1em 'Font Awesome 5 Free';
         content: "\F078";
```

Comment:
This doesn't show up for me on my local build 
![screen shot 2019-01-25 at 3 16 22 pm](https://user-images.githubusercontent.com/11021491/51770789-e0d88880-20b4-11e9-8c1b-eff811ef0763.png)